bin/pycheck

```diff
@@ -45,6 +45,10 @@ fi
 
 try "$root"/bin/pyactivate --dev -m mypy --pretty "${py_files[@]}"
 
+(
+    cd misc/python
+    try "$root"/bin/pyactivate --dev -m pytest -q
```

Comment:
This may one day be too expensive to run in `bin/lint`; we'll have to keep an eye on it.

Reply:
strong agree!